docs/setting-up/server/helm.md

```diff
@@ -122,8 +163,65 @@ certs:
     dhparam.pem: <content>
 ```
 
-### Upgrades
+Another approach to set up TLS certificates is to use the Ingress controller, see [TLS](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls). PMM helm chart supports Ingress. See [PMM network configuration](https://github.com/percona/percona-helm-charts/tree/main/charts/pmm#pmm-network-configuration).
+
+## Backup
+
+PMM helm chart uses [PersistentVolume and PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to allocate storage in the Kubernetes cluster.
 
+Volumes could be pre-provisioned and dynamic. PMM chart supports both and exposes it through [PMM storage configuration](https://github.com/percona/percona-helm-charts/tree/main/charts/pmm#pmm-storage-configuration).
+
+Backups for the PMM server currently support only storage layer backups and thus require [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) and [VolumeSnapshotClass](https://kubernetes.io/docs/concepts/storage/volume-snapshot-classes/).
+
+Validate the correct configuration by using these commands:
+```sh
+kubectl get sc
+kubectl get volumesnapshotclass
+```
+
+!!! note alert alert-primary "Storage"
+    Storage configuration is Hardware and Cloud specific. There could be additional costs associated with Volume Snapshots. Check the documentation for your Cloud or for your Kubernetes cluster.
+
+Before taking a [VolumeSnapshot](https://kubernetes.io/docs/concepts/storage/volume-snapshots/), stop the PMM server. In this step, we will stop PMM (scale to 0 pods), take a snapshot, wait until the snapshot completes, then start PMM server (scale to 1 pod):
+```sh
+kubectl scale statefulset pmm --replicas=0
+kubectl wait --for=jsonpath='{.status.replicas}'=0 statefulset pmm
+
+cat <<EOF | kubectl create -f -
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: before-v2.34.0-upgrade
+  labels:
+    app.kubernetes.io/name: "pmm"
```

Comment:
Purely on a standardisation basis... but, why is it double-quoted and nothing else is and this does not contain anything that would need it to be quoted either?

Reply:
copy-paste from k8s output :)

Reply:
Always review what you c&p :)